nucleus/osgi-platforms/felix/src/main/resources/config/osgi.properties

```diff
@@ -183,14 +183,18 @@ glassfish.osgi.auto.install=\
 hk2.bundles=\
  ${com.sun.aas.installRootURI}modules/hk2-api.jar \
  ${com.sun.aas.installRootURI}modules/hk2-utils.jar \
- ${com.sun.aas.installRootURI}modules/javax.inject.jar \
+ ${com.sun.aas.installRootURI}modules/jakarta.inject.jar \
```

Comment:
Is this related?

Reply:
Yes, HK2 2.6 depends on Jakarta injection